### DISCUSSION TITLE
Tools to handle the csv files?

### DISCUSSION THREAD

**Discussion by gzclarence** [TulipaEnergy/TulipaEnergyModel.jl#275] [Questions] [answered]
Hi there!

I am building the EU case. Right now I am using pandas in Python to edit the csv. Any suggestions on how/with what tool I can input data more easily? What are you using for this?

Thanks in advance,
Zhi

**Comment by datejada** [comment, marked as answer]
Hi @gzclarence,

So far, we are still looking for a better way to work with the CSVs. We have some options, but this will take time, so I suggest using the programming language and packages you feel more confident in (Python/Pandas if that is the case). If you want to start working with Julia and have the time, you can explore the CSV and DataFrames packages in Julia for data wrangling.

Best,

Diego
